sampleContract/src/my-private-asset-contract.ts

```diff
@@ -71,23 +71,23 @@ export class MyPrivateAssetContract extends Contract {
         if (!exists) {
             throw new Error(`The private asset ${myPrivateAssetId} does not exist`);
         }
-        await ctx.stub.deletePrivateData(myCollection, myPrivateAssetId);
+        await ctx.stub.deletePrivateData(myCollectionName, myPrivateAssetId);
     }
 
     @Transaction(false)
-    public async verifyMyPrivateAsset(ctx: Context, collection: string, myPrivateAssetId: string, hashToVerify: string): Promise<string> {
+    public async verifyMyPrivateAsset(ctx: Context, myPrivateAssetId: string, hashToVerify: string): Promise<boolean> {
 
-        const pdHashBytes: Buffer = await ctx.stub.getPrivateDataHash(collection, myPrivateAssetId);
+        const pdHashBytes: Buffer = await ctx.stub.getPrivateDataHash(myCollectionName, myPrivateAssetId);
         if (pdHashBytes.length === 0) {
             throw new Error('No private data hash with the key: ' + myPrivateAssetId);
         }
 
         const actualHash: string = pdHashBytes.toString('hex');
 
         if (hashToVerify === actualHash) {
-            return '\nCalculated Hash provided: \n' + hashToVerify + '\n\n             MATCHES  <----->  \n\nHash from Private Data Hash State \n' + actualHash;
+            return true;
         } else {
-            throw new Error('Could not match the Private Data Hash State');
+            throw new Error('Could not match the Private Data Hash State. Please try again.');
```

Comment:
Can we change this error message to "No match found for ${provided-hash}" or something like that? (you'll have to update your tests too)